tst/Encamina.Enmarcha.AI.Tests/Encamina.Enmarcha.AI.Tests.csproj

```diff
@@ -7,5 +7,35 @@
     <ItemGroup>
       <ProjectReference Include="..\..\src\Encamina.Enmarcha.AI\Encamina.Enmarcha.AI.csproj" />
     </ItemGroup>
+
```

Comment:
This all configurations are in a `Directory.Build.targets` at Test level.

![image](https://github.com/user-attachments/assets/ab75dc42-0372-496b-855e-06fc135d2ddf)